src/schema.ts

```diff
@@ -4,1283 +4,1283 @@
+    head?: never;
+    patch?: never;
+    trace?: never;
+  };
+  '/v2/fees/transfer': {
+    parameters: {
+      query?: never;
+      header?: never;
+      path?: never;
+      cookie?: never;
+    };
+    /**
+     * Get estimated fee
+     * @description Get an estimated fee rate for STX transfer transactions. This a a fee rate / byte, and is returned as a JSON integer
+     */
+    get: operations['get_fee_transfer'];
+    put?: never;
+    post?: never;
+    delete?: never;
+    options?: never;
+    head?: never;
+    patch?: never;
+    trace?: never;
+  };
+  '/v2/info': {
+    parameters: {
+      query?: never;
+      header?: never;
+      path?: never;
+      cookie?: never;
+    };
+    /**
+     * Get Core API info
+     * @description Get Core API information
+     */
+    get: operations['get_core_api_info'];
+    put?: never;
+    post?: never;
+    delete?: never;
+    options?: never;
+    head?: never;
+    patch?: never;
+    trace?: never;
+  };
+  '/v2/pox': {
+    parameters: {
+      query?: never;
+      header?: never;
+      path?: never;
+      cookie?: never;
+    };
+    /**
+     * Get PoX details
+     * @description Get Proof of Transfer (PoX) information. Can be used for Stacking.
+     */
+    get: operations['get_pox_info'];
+    put?: never;
+    post?: never;
+    delete?: never;
+    options?: never;
+    head?: never;
+    patch?: never;
+    trace?: never;
+  };
+  '/v2/traits/{contract_address}/{contract_name}/{trait_contract_address}/{trait_contract_name}/{trait_name}': {
+    parameters: {
+      query?: {
+        /** @description The Stacks chain tip to query from.
+         *     If tip == "latest", the query will be run from the latest known tip (includes unconfirmed state).
+         *     If the tip is left unspecified, the stacks chain tip will be selected (only includes confirmed state).
+         *      */
+        tip?: string;
+      };
+      header?: never;
+      path: {
+        /** @description Stacks address */
+        contract_address: string;
+        /** @description Contract name */
+        contract_name: string;
+        /** @description Trait Stacks address */
+        trait_contract_address: string;
+        /** @description Trait contract name */
+        trait_contract_name: string;
+        /** @description Trait name */
+        trait_name: string;
+      };
+      cookie?: never;
+    };
+    /**
+     * Get trait implementation details
+     * @description Determine whether or not a specified trait is implemented (either explicitly or implicitly) within a given contract.
+     */
+    get: operations['get_is_trait_implemented'];
+    put?: never;
+    post?: never;
+    delete?: never;
+    options?: never;
+    head?: never;
+    patch?: never;
+    trace?: never;
+  };
+  '/v2/constant_val/{contract_address}/{contract_name}/{constant_name}': {
+    parameters: {
+      query?: never;
+      header?: never;
+      path?: never;
+      cookie?: never;
```

Comment:
Upstream openapi schema is so limited/broken. Thoughts on using an improved local copy for now which we PR to upstream?

Reply:
Sure we can maintain something. Probably easier than trying to regularly PR into core